core/tests/ts-integration/tests/ether.test.ts

```diff
@@ -65,8 +81,8 @@ describe('ETH token checks', () => {
             amount,
             gasPerPubdataByte,
             l2GasLimit,
-            approveBaseERC20: true,
-            approveERC20: true,
+            approveBaseERC20: isETHBasedChain ? true : false,
```

Comment:
strange - why do we have to approveBaseERC20 when using ETH based chain?

Reply:
Test already had this flag set to `true` so I left it as it was just in case it made any change when using an ETH based chain. Further inspecting the sdk, it seems like `approbeBaseERC20`'s value is not taken into account at all when doing a [_depositETHToETHBasedChain](https://github.com/zksync-sdk/zksync-ethers/blob/7c7fd450f5dca263796b687001f888989bd64ac9/src/adapters.ts#L645-L670). Considering this, the line can be entirely removed.